### DISCUSSION TITLE
How to build a Developer Profile - HugoConf2022 | ashish.one

### DISCUSSION THREAD

**Discussion by giscus** [ashishtiwari1993/ashish.one#21] [General]
# How to build a Developer Profile - HugoConf2022 | ashish.one

Just a thought 💭 … As a Developer, we keep learning & developing stuff. Sometimes we also find the solutions. Our stuff is distributed the same as our System architecture. We maintain different platforms like GitHub for projects, Medium for blogs, LinkedIn for profile, etc.
All this stuff we want to share on a single platform but as a tech, I am lazy if you ask me to install some CMS and maintain all stuff over there.

https://ashish.one/blogs/build-developer-profile/

**Comment by abhishek-one** [comment]
Very helpful. I was searching for this.
